usaspending_api/download/tests/unit/test_file_generation.py

```diff
@@ -127,119 +127,3 @@ def test_idv_treasury_account_funding_csv_sources():
     VALUE_MAPPINGS["idv_federal_account_funding"]["filter_function"] = original
     assert csv_sources[0].file_type == "treasury_account"
     assert csv_sources[0].source_type == "idv_federal_account_funding"
-
-
-def test_apply_annotations_to_sql_just_values():
```

Comment:
I'm worried that these tests were removed. Shouldn't they still pass?

Reply:
The sample SQL was not of the format that Django spits out, which the parser relies on. A new test with more representative SQL was added to the new test class, in order to have all annotation tests in the same place. These tests have nothing to do with generating a file.

Reply:
If it's that fragile and tied to Django, are you concerned with the near-future upgrade to Django 2.X?
https://federal-spending-transparency.atlassian.net/browse/DEV-3074